<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaInventario</class>
 <widget class="QMainWindow" name="ventanaInventario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="listaInventario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1901</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelnventario">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>180</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INVENTARIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBuscar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineBuscarProducto">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarProducto">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>910</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agregar Producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificarProducto">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>910</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modificar Producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarCategoria">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>910</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agregar Categoria</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminarProducto">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>910</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar Producto</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
